src/main/java/examples/CassandraClientExamples.java

```diff
@@ -113,6 +115,40 @@ public void streaming(CassandraClient cassandraClient) {
     });
   }
 
+  public void streamingViaHttp(Vertx vertx, CassandraClient cassandraClient) {
```

Comment:
Ha. I was thinking about having just one example, `streaming`. We don't need to show how to create a http server and handle requests, you may simply put the `HttpServerResponse` object as a method param.

Here we have two long similar examples

Reply:
Agree, done